electrum/gui/qt/transaction_dialog.py

```diff
@@ -476,8 +480,10 @@ def update(self):
             amount_str = ''
         elif amount > 0:
             amount_str = _("Amount received:") + ' %s'% format_amount(amount) + ' ' + base_unit
+            amount_str += ' %s'%  format_fiat_and_units(amount)
         else:
             amount_str = _("Amount sent:") + ' %s'% format_amount(-amount) + ' ' + base_unit
+            amount_str += ' %s'%  format_fiat_and_units(-amount)
```

Comment:
This field is visible not only in the "Create Transaction" dialog (advanced preview), but also in the "Transaction Details" dialog (when inspecting old txs in history). In such cases, this would use the incorrect FX conversion rate (should use historical rate instead of current rate).